app/org/sagebionetworks/bridge/dynamodb/DynamoSubpopulationDao.java

```diff
@@ -91,6 +94,12 @@ public Subpopulation updateSubpopulation(Subpopulation subpop) {
         DynamoDBQueryExpression<DynamoSubpopulation> query = 
                 new DynamoDBQueryExpression<DynamoSubpopulation>().withHashKeyValues(hashKey);
         
+        if (!includeDeleted) {
+            query.withQueryFilterEntry("deleted", new Condition()
+               .withComparisonOperator(ComparisonOperator.EQ)
+               .withAttributeValueList(new AttributeValue().withN("0")));
+        }
+
```

Comment:
Minor: Not sure if this is worth the extra code churn. The savings we get in CPU and network are miniscule, and this won't allow us to save DDB capacity. Recommend reverting this.

I also noticed that there's no test for includeDeleted=true.